alerter/multikustoclient/mulikistoclient.go

```diff
@@ -18,8 +19,10 @@ type multiKustoClient struct {
 func New(endpoints map[string]string, configureAuth authConfiguror, max int) (multiKustoClient, error) {
 	clients := make(map[string]*kusto.Client)
 	for name, endpoint := range endpoints {
-		kcsb := kusto.NewConnectionStringBuilder(endpoint).WithAzCli()
-		kcsb = configureAuth(kcsb)
+		kcsb := kusto.NewConnectionStringBuilder(endpoint)
+		if strings.HasPrefix(endpoint, "https://") {
+			kcsb = configureAuth(kcsb.WithAzCli())
+		}
```

Comment:
This allows Alerter to connect to Kustainer for integration tests

Reply:
Is Kustotainer endpoint not using https?  Is that how this change makes it work with it?